tests/McRouterTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+require_once 'AbstractBackendTest.php';
+
+use iFixit\Matryoshka;
+
+class McRouterTest extends AbstractBackendTest {
```

Comment:
Let's extend `MemachedTest` instead so we don't have to implement the `testMcrouter` function (which is really just testing expiration)